packages/subgraph/tasks/deploy-to-hosted-service-network.sh

```diff
@@ -4,6 +4,12 @@
 
 graph="../../node_modules/@graphprotocol/graph-cli"
 
+# prepare the manifest prior to deployment
+# this generates the subgraph.yaml and
+# inputs the correct addresses for the specified network ($2)
+./tasks/prepare-manifest.sh $2
```

Comment:
ditto, would add a shellcheck lint as tech debt.

Reply:
I added a new `shellcheck-all-shell-scripts.sh` which checks all our bash scripts and it would run in `ci.lint-workflow-files.yml`.
It is currently commented out as it would cause shellcheck is not happy with a bunch of our shell scripts. This is just a brute force solution and can be tossed. 
The other solution is adding: `shell: bash` in all our `run:` commands in the different CI files.

Reply:
Actually, this is not the best:

you could add this to the lint step of the package.json

Reply:
And `find tasks $(find packages -name tasks) -name '*.sh'`

Reply:
Let me fix this

Reply:
one more workflow file is not necessary.

Reply:
I agree, I'd rather not add another shell script file and instead do a separate PR where we just do `shell: bash`.